src/SourceLocator/SourceStubber/AggregateSourceStubber.php

```diff
@@ -47,4 +47,20 @@ public function generateFunctionStub(string $functionName) : ?StubData
 
         return null;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function generateConstantStub(string $constantName) : ?StubData
+    {
+        foreach ($this->sourceStubbers as $sourceStubber) {
```

Comment:
Not important, but this stuff ia usually an `array_reduce()` with a `??` in its body

Reply:
Done.